qa/L0_infer/test.sh

```diff
@@ -130,7 +132,11 @@ for TARGET in cpu gpu; do
     for FW in $BACKENDS; do
       if [ "$FW" != "plan" ]; then
         for MC in `ls models/${FW}*/config.pbtxt`; do
-            echo "instance_group [ { kind: ${KIND} }]" >> $MC
+            if [ "$FW" == "custom" ]; then
```

Comment:
Doesn't L0_infer use the addsub custom backend? That backend does support both CPU and GPU.

Reply:
We built the custom backends (Dockerfile.custombackend) with TRTIS_ENABLE_GPU=OFF.
When we do this, the GPU instance is not supported.

Reply:
Fixed.